sdk/cosmos/azure-cosmos-tests/src/test/java/com/azure/cosmos/implementation/directconnectivity/JsonNodeStorePayloadTests.java

```diff
@@ -0,0 +1,46 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.cosmos.implementation.directconnectivity;
+
+import io.netty.buffer.ByteBuf;
+import io.netty.buffer.ByteBufInputStream;
+import io.netty.buffer.Unpooled;
+import org.testng.annotations.Ignore;
+import org.testng.annotations.Test;
+
+public class JsonNodeStorePayloadTests {
+    @Test(groups = {"unit"})
+    @Ignore("Need to figure out a way to reload the class")
```

Comment:
what is the issue here?

Reply:
discussed offline. Update the comment to make it more clear